src/otx/cli/cli.py

```diff
@@ -250,8 +267,24 @@ def add_subcommands(self) -> None:
             self._subcommand_parsers[subcommand] = sub_parser
             parser_subcommands.add_subcommand(subcommand, sub_parser, help=description)
 
-    def _set_default_config_from_auto_configurator(self) -> dict:
+    def _load_cache_ckpt(self, parser: ArgumentParser) -> None:
+        if (self.cache_dir / "latest_checkpoint.txt").exists():
+            with (self.cache_dir / "latest_checkpoint.txt").open("r") as f:
+                latest_checkpoint = f.read()
+            parser.set_defaults(checkpoint=latest_checkpoint)
+            if "--print_config" not in sys.argv:
+                warn(f"Load default checkpoint from {latest_checkpoint}.", stacklevel=0)
+
+    def _set_default_config(self) -> dict:
         parser_kwargs = {}
+        if (self.cache_dir / "configs.txt").exists():
```

Comment:
```suggestion
        if not (self.cache_dir / "configs.txt").exists():
             return
```

Reply:
I left this as is so that if don't have the latest, CLI can load a different default config. Instead, I added a comment below